cmd/auto-pause/auto-pause.go

```diff
@@ -55,7 +53,7 @@ func main() {
 			// On each iteration new timer is created
 			select {
 			// TODO: #10596 make it memory-leak proof
-			case <-time.After(interval):
+			case <-time.After(addons.AutoPauseTime):
```

Comment:
auto-pause runs on the guest, so the settings on the host will NOT propagate here. We need to pass this in through the command line arguments for auto-pause.

Reply:
Made it so that it should be configurable via the configurate command. @andriyDev